source/NVDAObjects/IAccessible/sysListView32.py

```diff
@@ -326,14 +330,28 @@ def _getColumnContentRaw(self, index):
 					winKernel.readProcessMemory(processHandle,internalItem,byref(item),sizeof(self.LVITEM),None)
 					buffer=create_unicode_buffer(len)
 					winKernel.readProcessMemory(processHandle,item.pszText,buffer,sizeof(buffer),None)
+			except ArgumentError as e:
+				log.debugWarning("%r,\nCan't retrieve item text, switching to display text instead" % e)
+				self.parent._shouldEnableColumnContentRaw = False
+				buffer = None
 			finally:
 				winKernel.virtualFreeEx(processHandle,internalText,0,winKernel.MEM_RELEASE)
 		finally:
 			winKernel.virtualFreeEx(processHandle,internalItem,0,winKernel.MEM_RELEASE)
 		return buffer.value if buffer else None
 
 	def _getColumnContent(self, column):
-		return self._getColumnContentRaw(self.parent._columnOrderArray[column - 1])
+		res = None
+		if self.parent._shouldEnableColumnContentRaw:

```

Comment:
Just to make sure that this doesn't horribly crash if the parent is not a list and therefore lacks this property, could you change this to:

`if getattr(self.parent, "_shouldEnableColumnContentRaw", True):`

I agree that this is not very elegant, however a traceback ain't of very much elegance either.